.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '18.x'
+      - name: Print versions
+        run: |
+          node --version
+          npm --version
+          npx commitlint --version
+      - name: Install commitlint
+        run: |
+          npm install -g commitlint
+      - name: Validate PR commits with @commitlint/cli
+        if: github.event_name == 'pull_request'
```

Comment:
Why are we checking this at such a late point in time? (Also not sure how it works exactly/whether we actually need it?)

Reply:
thanks. this is redundant as we're limiting this pipeline to run only on `pull_request` anyway (see above)